src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -3347,20 +3346,9 @@ Here is example using a few of the rsgroup  commands. To add a group, do as foll
 .RegionServer Groups must be Enabled
 [NOTE]
 ====
-If you have not enabled the rsgroup Coprocessor Endpoint in the master and
-you run the any of the rsgroup shell commands, you will see an error message
-like the below:
-
-[source,java]
-----
-ERROR: org.apache.hadoop.hbase.exceptions.UnknownProtocolException: No registered master coprocessor service found for name RSGroupAdminService
-    at org.apache.hadoop.hbase.master.MasterRpcServices.execMasterService(MasterRpcServices.java:604)
-    at org.apache.hadoop.hbase.shaded.protobuf.generated.MasterProtos$MasterService$2.callBlockingMethod(MasterProtos.java)
-    at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:1140)
-    at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
-    at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:277)
-    at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:257)
-----
+If you have not enabled the rsgroup in the master and you call the any of
+rsgroup admin methods, or run the any of the rsgroup shell commands, you may
+see a `DoNotRetryIOException` which tells that 'RSGroup is disabled'
```

Comment:
Does the `DoNotRetryIOException` make clear that the feature is off? or does it talk about some other thing that we can give an example of here to distinguish from some coincidental failure?

Reply:
I think the message "RSGroup is disabled" is clear enough? You mean add a "set 'hbase.balancer.rsgroup.enabled' to true if you want to use RSGroup"?

Reply:
that message would be great! instructions on how to turn it on would be even better, but I'd be happy with a clear statement that the feature is off as the cause.

I see now why I misunderstood. I took "see a X which tells that Y" to mean "you will see X and you should interpret it to mean Y" rather than the intended "you will see X and it will contain details that Y".

suggested edit for this sentence:
> If you have not enabled the rsgroup feature and you call any of the rsgroup admin methods or shell commands the call will fail with a `DoNotRetryIOException` with a detail message that says the rsgroup feature is disabled.

wdyt?

Reply:
Thanks. Let me change the words.